vllm/v1/worker/gpu_model_runner.py

```diff
@@ -883,6 +883,7 @@ def _calc_spec_decode_metadata(
             target_logits_indices=target_logits_indices,
             bonus_logits_indices=bonus_logits_indices,
             logits_indices=logits_indices,
+            total_num_scheduled_tokens=cu_num_scheduled_tokens[-1],
```

Comment:
Do we actually need to store it in `SpecDecodeMetadata`?
I'm wondering because the same variable is available in `execute_model`.